module/VuFind/tests/integration-tests/src/VuFindTest/Mink/ApiTest.php

```diff
@@ -40,18 +47,106 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Page
  */
-class ApiTest extends \VuFindTest\Integration\MinkTestCase
+final class ApiTest extends \VuFindTest\Integration\MinkTestCase
```

Comment:
I try to explain why classes are final when I make them so. You can copy-paste the comment from here for consistency: https://github.com/vufind-org/vufind/blob/e64fb7aad8847cc27e0a17bc4675be295ae80547/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LibraryCardsTest.php#L37